mods/tnt/init.lua

```diff
@@ -420,6 +422,7 @@ minetest.register_node("tnt:gunpowder", {
 		minetest.set_node(pos, {name = "tnt:gunpowder_burning"})
 	end,
 	on_ignite = function(pos, igniter)
+		minetest.sound_play("tnt_ignite", {pos = pos})
```

Comment:
You are right, I'm not sure why I put this here... if you didn't do it before, I'll remove it when I reach home in a few hours